src/test/suite/utilities.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as assert from 'assert';
+import { getRepositoryName } from '../../utilities';
+
+suite('Utilities Test Suite', () => {
```

Comment:
Guess we should be adding tests as a default, I've no idea how the test framework works in vscode though. Also would be good to add GitHub CI to run tests automatically on PRs (for another PR)

Reply:
I'll add this in a separate PR

Reply:
@0xTim I meant for test the vscode-swift typescript not adding support for testing swift

Reply:
Yeah CI for the extension itself